src/apis/axios.ts

```diff
@@ -0,0 +1,30 @@
+import axios, { AxiosInstance, AxiosResponse } from 'axios';
+
+const instance: AxiosInstance = axios.create({
+  baseURL: process.env.NEXT_PUBLIC_URL,
+});
+
+instance.interceptors.request.use(
+  (config) => {
+    const accessToken = sessionStorage.getItem('accessToken');
+    if (accessToken) {
+      config.headers = config.headers || {};
+      config.headers['Authorization'] = `Bearer ${accessToken}`;
+    }
```

Comment:
```suggestion
    } else {
       delete config.headers['Authorization'];
    }
```
accessToken이 없다면 Authorization를 삭제하는 구문이 들어가면 좋을 것 같습니다.

Reply:
✅ [57f2f63](https://github.com/PRism-Works/PRism-FE/pull/44/commits/57f2f63b1f28e94248fef4bd8aac551f326a44e7) 추가했습니다.